From 507d3070a2cb3b6a2ef2f54d9f9a39c1162b9a06 Mon Sep 17 00:00:00 2001
From: Yichao Yu <yyc1992@gmail.com>
Date: Thu, 30 Sep 2021 00:01:11 -0400
Subject: [PATCH 09/10] Allow a higher initial heap size in test

A `/bin/sleep` has an initial heap size fo 33 pages on my local setup.
-1 pages is way too small to fail.
---
 test/test-kkkkkkkk.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/test/test-kkkkkkkk.c b/test/test-kkkkkkkk.c
index 2747c11..d4b640d 100644
--- a/test/test-kkkkkkkk.c
+++ b/test/test-kkkkkkkk.c
@@ -50,7 +50,9 @@ int main()
 
 	test_title("Don't set the \"brk\" below its initial value");
 	test_brk(page_size, 1);
-	test_brk(-2 * page_size, 0);
+	// Try to shrink the size of the heap by more than it's current size
+	// and it should fail.
+	test_brk(-1000 * page_size, 0);
 	test_brk(-page_size, 1);
 	test_result();
 
-- 
2.33.0

